### ISSUE TITLE
Illegal instruction error on running test-install 

### ISSUE THREAD

**Issue by sakurakhadag** [allenai/allennlp#4040]
**System:**
 - OS: Linux
 - Python version: 3.7.7
 - AllenNLP version: v0.9.0 and v0.9.1
 - PyTorch version: v1.4.0

**Problem**
I'm getting an Illegal Instruction error in multiple files when running test-install. 

The problem seems to be the tensor types, and the error is always at the tensor.normal_() implementations. I tried to typecast to FloatTensor just to see if it would solve the problem, and it did. For one file.

I tried changing AllenNLP versions, but I'm getting the same error. 

Could you please help?

The complete error is
```
allennlp/tests/version_test.py ...                                                                               [  0%]
allennlp/tests/commands/docstring_help_test.py .                                                                 [  0%]
allennlp/tests/commands/elmo_test.py Fatal Python error: Illegal instruction

Current thread 0x00007fddee6ad700 (most recent call first):
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/torch/nn/init.py", line 376 in orthogonal_
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/nn/initializers.py", line 139 in block_orthogonal
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/modules/lstm_cell_with_projection.py", line 90 in reset_parameters
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/modules/lstm_cell_with_projection.py", line 86 in __init__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/modules/elmo_lstm.py", line 92 in __init__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/modules/elmo.py", line 541 in __init__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/commands/elmo.py", line 201 in __init__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/commands/elmo.py", line 410 in elmo_command
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/commands/__init__.py", line 93 in main
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/tests/commands/elmo_test.py", line 47 in test_all_embedding_works
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/python.py", line 184 in pytest_pyfunc_call
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/unittest.py", line 220 in wrapped_testMethod
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/unittest/case.py", line 628 in run
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/unittest/case.py", line 676 in __call__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/unittest.py", line 232 in runtest
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/runner.py", line 135 in pytest_runtest_call
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/runner.py", line 217 in <lambda>
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/runner.py", line 244 in from_call
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/runner.py", line 217 in call_runtest_hook
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/flaky/flaky_pytest_plugin.py", line 129 in call_and_report
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/runner.py", line 100 in runtestprotocol
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/runner.py", line 85 in pytest_runtest_protocol
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/flaky/flaky_pytest_plugin.py", line 92 in pytest_runtest_protocol
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/main.py", line 272 in pytest_runtestloop
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/main.py", line 247 in _main
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/main.py", line 191 in wrap_session
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/main.py", line 240 in pytest_cmdline_main
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/_pytest/config/__init__.py", line 125 in main
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/commands/test_install.py", line 84 in _run_test
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/commands/__init__.py", line 93 in main
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/lib/python3.7/site-packages/allennlp/__main__.py", line 19 in run
  File "/home/ishitatripathi/miniconda3/envs/allennlp9.1/bin/allennlp", line 8 in <module>
Illegal instruction
```

**Comment by schmmd** [comment]
It's likely an incompatibility issue with torch.  When we first released we were using 1.2.0 for 0.9.0: https://github.com/allenai/allennlp/blob/052353ed62e3a54fd7b39a660e65fc5dd2f91c7d/requirements.txt#L12

**Comment by sakurakhadag** [comment]
So I built everything again today. It seems to be magically working. PyTorch has updated to v1.5. Maybe they fixed the problem. I'll close this thread now.
